htlcswitch/switch.go

```diff
@@ -1932,6 +1940,18 @@ func (s *Switch) AddLink(link ChannelLink) error {
 		)
 	}
 
+	// Since the link addition has succeeded, this channel is now officially
+	// active and we can tell the ChannelNotifier about it.
+	peerPub := link.Peer().PubKey()
+	channel, err := s.cfg.DB.FetchOpenChannel(peerPub[:], link.ChannelPoint())
+	if err != nil {
+		log.Warnf("errored fetching channel with channelpoint=%v: %v",
```

Comment:
Just for consistency, we typically use present tense in most log msgs, e.g. "Unable to fetch"